src/java/htsjdk/variant/variantcontext/GenotypeLikelihoods.java

```diff
@@ -338,6 +351,29 @@ public GenotypeLikelihoodsAllelePair(final int alleleIndex1, final int alleleInd
         return cache;
     }
 
+
+    /**
+     * Calculate the cache of anyploid alleles for each PL index
+     * Creates the ordering for all possible combinations of ploidy alleles. Computed recursively and the
+     * result is stored in a cache.
```

Comment:
could you include the recursive formula you are using and a reference for where it comes from?